### ISSUE TITLE
Your first issue

### ISSUE THREAD

**Issue by crispg72** [bezzy199991/Hi#1]
Hey you 😉 

Nice, I see what you've done and well done for contributing 👍 

Going for lunch now but I'll add some comments in a bit

**Comment by bezzy199991** [comment]
Okay cool thank you :)

**Comment by crispg72** [comment]
Right, I see you created the code in `README.md`.....try creating a python file instead so that when I clone this repo I can just run it.

Like [this file](https://github.com/geekcomputers/Python/blob/master/google.py) for example - I chose this at random. Copy your code into `hello_world.py` or something similar and add it to this repo.
